src/cpp/lpnamer/main/ProblemGeneration.cpp

```diff
@@ -53,6 +56,20 @@ ProblemGeneration::ProblemGeneration(ProblemGenerationOptions& options)
   }
 }
 
+std::filesystem::path ProblemGeneration::performAntaresSimulation() {
+  auto results = Antares::API::PerformSimulation(options_.StudyPath());
+  //Add parallel
+
+  //Handle errors
+  if (results.error) {
+    std::cerr << "Error: " << results.error->reason << std::endl;
```

Comment:
use the custom logger

Reply:
https://github.com/AntaresSimulatorTeam/antares-xpansion/issues/888